src/Screens/HomeScreen/HomeScreen.tsx

```diff
@@ -25,9 +23,12 @@ const HomeScreen = () => {
 
   const alert = useAlert();
 
-  const { loading } = useSelector((state: AppState) => state.todosReducer);
-
+  const { loading, todos } = useSelector(
+    (state: AppState) => state.todosReducer
+  );
 
+  const activeTodos = todos.filter((todo) => todo.completed === false);
```

Comment:
should be enough ->   const activeTodos = todos.filter((todo) => !todo.completed);